nginx/nginx.go

```diff
@@ -208,8 +206,6 @@ func (n *nginxUpdater) ensureNginxRunning() error {
 
 		go n.periodicallyUpdateMetrics()
 		go n.backgroundSignaller()
-
-		n.initialUpdateApplied.done = true
```

Comment:
How come this was moved? If we move it, it won't try to restart nginx if it fails due to bad config (maybe that's okay, though - kubelet will kill it eventually).